src/Compilers/CSharp/Portable/Binder/BinderFactory.BinderFactoryVisitor.cs

```diff
@@ -684,6 +684,12 @@ private Binder VisitTypeDeclarationCore(TypeDeclarationSyntax parent)
                 {
                     extraInfo = NodeUsage.NamedTypeBodyOrTypeParameters;
                 }
+                else if (parent.SemicolonToken != default &&
+                    parent is RecordDeclarationSyntax &&
+                    LookupPosition.IsBetweenTokens(_position, parent.GetFirstToken(), parent.SemicolonToken.GetNextToken()))
+                {
+                    extraInfo = NodeUsage.NamedTypeBodyOrTypeParameters;
+                }
```

Comment:
This change fixes `cref="SomePositionalMember"` on the record. It makes the cref binder be an InContainerBinder for the record. I'm not aware if this has any other effects (and whether these effects are desired or not)